index.html

```diff
@@ -3515,10 +3515,10 @@ <h5>JWT Encoding</h5>
             </p>
 
             <p>
-If a JWS is present, the digital signature either refers to the <a>issuer</a>
+If a JWS is present, the digital signature refers to either the <a>issuer</a>
 of the <a>verifiable credential</a>, or in the case of a
-<a>verifiable presentation</a>, the <a>holder</a> of the
-<a>verifiable credential</a>. The JWS proves that the <a>issuer</a> of the JWT
+<a>verifiable presentation</a>, to the <a>holder</a> of the
+<a>verifiable credential</a>. The JWS proves that the <a>iss</a> of the JWT
```

Comment:
I think here `<a>issuer</a>` is more correct, since the sentence is talking about the concept of an issuer rather than a concrete field in an object.

Or alternatively, change to `<code>iss</code>` instead of `<a>iss</a>`, since "iss" is not en entry in the terminology.